controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -563,8 +563,8 @@ void cli_bml::setFunctionsMapAndArray()
         " selected_bands - Bitwise parameter, 1 for 2.4G, 2 for 5G, 3 for both, 0 for Disabled"
         " stay_on_initial_radio - 1 for true, 0 for false or (default) -1 for not configured,"
         " stay_on_selected_device - 1 for true, 0 for false or (default) -1 for not configured",
-        static_cast<pFunction>(&cli_bml::client_set_client_caller), 2, 4, STRING_ARG, INT_ARG,
-        INT_ARG, INT_ARG);
+        static_cast<pFunction>(&cli_bml::client_set_client_caller), 2, 4, STRING_ARG, STRING_ARG,
```

Comment:
1. commit message:
- change `beerocks_cli` to `beerocks_cli_bml`
- please explain on each change why the changes are needed

Reply:
Changed